mystery-deckbuilder/Assets/MainMenu.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+
+public class MainMenu : MonoBehaviour
+{
+    public void play() 
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex +1);
+    }
+
+    public void quit()
+    {
+        Debug.Log("Quit");
+        Application.Quit();
+    }
+}
```

Comment:
1. As per style guide, methods "play" and "quit" should be "Play" and "Quit"
2. As per practice guide, functions Play and Quit should be multiple-line commented

ex:
```csharp

/*  This method launches the MainMenu scene found in Assets/Scenes/MainMenu */
public void Play() 
{
   // the actual code goes here
}
```